src/VisualStudio/VisualBasic/Impl/Options/IntelliSenseOptionPageControl.xaml

```diff
@@ -15,14 +15,20 @@
             <GroupBox x:Uid="CompletionListsGroupBox"
                       Header="{x:Static local:IntelliSenseOptionPageStrings.Option_CompletionLists}">
                 <StackPanel>
-
                     <CheckBox x:Uid="Highlight_matching_portions_of_completion_list_items"
                                   x:Name="Highlight_matching_portions_of_completion_list_items"
                                   Content="{x:Static local:IntelliSenseOptionPageStrings.Option_Highlight_matching_portions_of_completion_list_items}" />
                     <CheckBox x:Uid="Show_completion_item_filters"
                                   x:Name="Show_completion_item_filters"
                                   Content="{x:Static local:IntelliSenseOptionPageStrings.Option_Show_completion_item_filters}" />
-
+                </StackPanel>
+            </GroupBox>
+            <GroupBox x:Uid="SelectionInCompletionListGroupBox"
+                     Header="{x:Static local:IntelliSenseOptionPageStrings.Option_SelectionInCompletionList}">
+                <StackPanel>
+                    <RadioButton  GroupName="InsertNewlineOnEnterRadio" x:Name="Never_insert_new_line" Content="{x:Static local:IntelliSenseOptionPageStrings.Option_NeverInsertNewlineOnEnter}"/>
```

Comment:
inconsistent names.  in one place you have Never_insert_new_line and right after that you have Option_NeverInsertNewlineOnEnter.  These names should be as consistent as possible across all the places you have to touch.